usr/local/www/diag_logs_settings.php

```diff
@@ -254,234 +251,302 @@ function check_everything() {
+
+$logfilesizeHelp =	gettext("Logs are held in constant-size circular log files. This field controls how large each log file is, and thus how many entries may exist inside the log. By default this is approximately 500KB per log file, and there are nearly 20 such log files.") .
+					'<br /><br />' .
+					gettext("NOTE: Log sizes are changed the next time a log file is cleared or deleted. To immediately increase the size of the log files, you must first save the options to set the size, then clear all logs using the \"Reset Log Files\" option farther down this page. ") .
+					gettext("Be aware that increasing this value increases every log file size, so disk usage will increase significantly.") . '<br /><br />' .
+					gettext("Disk space currently used by log files is: ") . exec("/usr/bin/du -sh /var/log | /usr/bin/awk '{print $1;}'") .
+					gettext(" Remaining disk space for log files: ") . exec("/bin/df -h /var/log | /usr/bin/awk '{print $4;}'");
+
+$remoteloghelp =	gettext("This option will allow the logging daemon to bind to a single IP address, rather than all IP addresses.") .
+					gettext("If you pick a single IP, remote syslog severs must all be of that IP type. If you wish to mix IPv4 and IPv6 remote syslog servers, you must bind to all interfaces.") .
+					"<br /><br />" .
+					gettext("NOTE: If an IP address cannot be located on the chosen interface, the daemon will bind to all addresses.");
+if ($input_errors)
+	print_input_errors($input_errors);
+else if($savemsg)
+	print('<div class="alert alert-success" role="alert">'.$savemsg.'</div>');
+
+$tab_array = array();
+$tab_array[] = array(gettext("System"), false, "diag_logs.php");
+$tab_array[] = array(gettext("Firewall"), false, "diag_logs_filter.php");
+$tab_array[] = array(gettext("DHCP"), false, "diag_logs_dhcp.php");
+$tab_array[] = array(gettext("Portal Auth"), false, "diag_logs_auth.php");
+$tab_array[] = array(gettext("IPsec"), false, "diag_logs_ipsec.php");
+$tab_array[] = array(gettext("PPP"), false, "diag_logs_ppp.php");
+$tab_array[] = array(gettext("VPN"), false, "diag_logs_vpn.php");
+$tab_array[] = array(gettext("Load Balancer"), false, "diag_logs_relayd.php");
+$tab_array[] = array(gettext("OpenVPN"), false, "diag_logs_openvpn.php");
+$tab_array[] = array(gettext("NTP"), false, "diag_logs_ntpd.php");
+$tab_array[] = array(gettext("Settings"), true, "diag_logs_settings.php");
+display_top_tabs($tab_array);
+
+require('classes/Form.class.php');
+
+$form = new Form(new Form_Button(
+	'Submit',
+	gettext("Save")
+));
+
+// General logging section ------------------------------------------------
+$section = new Form_Section('General Logging Options');
+
+$section->addInput(new Form_Checkbox(
+	'reverse',
+	'Forward/Reverse Display',
+	'Show log entries in reverse order (newest entries on top)',
+	$pconfig['reverse']
+));
+
+$section->addInput(new Form_Input(
+	'nentries',
+	'GUI Log Entries',
+	'text',
+	htmlspecialchars($pconfig['nentries']),
```

Comment:
That makes perfect sense. Thanks again for your guidance.

Steve


> On Apr 25, 2015, at 8:22 AM, SjonHortensius <notifications@github.com> wrote:
> 
> In usr/local/www/diag_logs_settings.php <https://github.com/SjonHortensius/pfsense/pull/134#discussion_r29100192>:
> 
> > +
> > +// General logging section ------------------------------------------------
> > +$section = new Form_Section('General Logging Options');
> > +
> > +$section->addInput(new Form_Checkbox(
> > +	'reverse',
> > +	'Forward/Reverse Display',
> > +	'Show log entries in reverse order (newest entries on top)',
> > +	$pconfig['reverse']
> > +));
> > +
> > +$section->addInput(new Form_Input(
> > +	'nentries',
> > +	'GUI Log Entries',
> > +	'text',
> > +	htmlspecialchars($pconfig['nentries']),
> Please remove htmlspecialchars to all Form calls
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/SjonHortensius/pfsense/pull/134/files#r29100192>.
> 

Steve
--
Stephen Beaver | +1 (512) 646-4100;239 | sbeaver@netgate.com <mailto:sbeaver@netgate.com> | www.netgate.com <http://www.netgate.com/>